.github/workflows/02_push.yml

```diff
@@ -1,4 +1,4 @@
-name: push
+name: cicd
```

Comment:
Why this rename? It runs on push and that's its distinction from the 01_pr for example. So I don't see `cicd` as a better name

And if it were a better name, the file name should also reflect this

Reply:
> Fixed and updated the workflow status badges in the README. Also renamed the workflow name for pushing events from 'push' to 'cicd' since this is represented in the status badge and would sound confusing.

Reply:
But all other files have same name and name attribute value, so this sticks out, would then at least add a comment here to explain this discrepancy